docker/all-in-one/entrypoint.sh

```diff
@@ -80,10 +80,23 @@ function enable_lsn_checkpoint_push {
 }
 
 function disable_fail2ban {
+  sed -i "s/command=.*/command=sleep 1/" /etc/supervisor/services/fail2ban.conf
```

Comment:
there is a danger here that this sleep isn't long enough. 
By default if a program doesn't stay up for 1 second, it is considered a failure and it will restart
I would set this to 2, or set startsecs to 0 - but startsecs isn't in the config file rn

from the manual

```
startsecs

The total number of seconds which the program needs to stay running after a startup to consider the start successful (moving the process from the STARTING state to the RUNNING state). Set to 0 to indicate that the program needn’t stay running for any particular amount of time.

Note

Even if a process exits with an “expected” exit code (see exitcodes), the start will still be considered a failure if the process exits quicker than startsecs.

Default: 1

Required: No.

Introduced: 3.0
```

Reply:
Ooh, nice catch. Will set it to `5` just to make sure it doesn't have any side effects